api-guidelines/v4.0/API-DESIGN-GUIDELINES/QUERY-OPERATORS.md

```diff
@@ -1,37 +1,39 @@
 # Query Operators
 
-An Ed-Fi API should support searching capabilities with the possible use of
-selectors, paging, and views. These are discussed below.
+An Ed-Fi API _should_ support searching capabilities with the possible use of filters, paging,
+and ordering. These are discussed below.  Information about all supported capabilities, their configurations, and their default values
+should be available from the API Metadata. <br /><br />
+When querying, an Ed-Fi API must use just the property value as the natural key query term, for example rather than schoolResource.schoolId
+the query term must be ?schoolId=xyz. 
 
 ## Search
 
 An Ed-Fi API _should_ support querying capabilities when searching a collection
 of Resources. Query operators are applied to the query string using the
-following format: `{collectionURI}?{propertyName}{operator}{value}`. Currently,
-the equals operator is the only operator specified. Other operators (>, <,
-'like', 'and', 'or', etc.) maybe implemented.
+following format: `{collectionURI}?{propertyName}={value}`. Additional `{propertyName}={value}` terms may be appended after an ampersand, &.
 
-**Table 4.** Equals Operator, the Only Operator Specified for Searching a
+For example, to search all available Students having the first name "John" and surname "Smith":
+
+```none
+https://api.example.com/v3/ed-fi/students?firstName=john&lastSurname=Smith
+```
+
+Not all properties are guaranteed to be searchable.  The Open API specification for the Ed-Fi API instance will define which properties are available.  All serachable properties in embedded objects must be dereferenced.  For example, a Student School Association document contains schoolReference: `{ schoolId: 12345 }`.  The equivalent query must be `/ed-fi/studentSchoolAssociations?schoolId=12345` not `/ed-fi/studentSchoolAssociations?schoolReference.schoolId=12345` 
```

Comment:
Reading this again now, I think "dereferenced" is used incorrectly here. It would actually mean the opposite of what we have. Let's rewrite this paragraph with something like this:

```
The Open API specification documents will show which properties are searchable.
When an a searchable parameter is nested under another object in the JSON
payload, the Ed-Fi API specifications require the query string to use only the
parameter name _without_ dereferencing the parent element name. For example, the
JSON document for a `studentSchoolAssociation` includes `schoolId` nested below
`schoolReference`. In the JsonPath query language, the query for a particular
schoolId would then be `?schoolReference.schoolId=xyz`. However, the Ed-Fi API
specifications predate and do not use JsonPath. Instead, they make a simplifying
assumption that `schoolId` by itself is uniquely searchable without
dereferencing the parent entity, `schoolReference`. Thus, the correct HTTP query
term is therefore simply `?schoolId=xyz`.
```